CSharp/Library/Microsoft.Bot.Connector.Shared/ConnectorAPI/Models/Activity.cs

```diff
@@ -133,13 +133,13 @@ public Activity(string type = default(string), string id = default(string), Date
         /// Array of address added
         /// </summary>
         [JsonProperty(PropertyName = "membersAdded")]
-        public IList<ChannelAccount> MembersAdded { get; set; }
+        public IList<ChannelAccount> MembersAdded { get; set; } = new ChannelAccount[0];

```

Comment:
not available in .Net Core. Would `Enumerable.Empty<T>().ToList()` be acceptable? seems like too much work.